build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -3,155 +3,16 @@
-        <Version>4.0.11</Version>
-        <TFM>netstandard1.3</TFM>
-        <DestinationTFM>net46</DestinationTFM>
-      </CopyLocalAssembly>
-
-
-      <CopyLocalAssembly>
-        <PackageRelativePath>$([System.String]::Copy('%(Identity)\%(Version)').ToLowerInvariant())</PackageRelativePath>
-      </CopyLocalAssembly>
-
-      <CopyLocalAssembly>
-        <FullFilePath>$(NuGet_Packages)\%(CopyLocalAssembly.PackageRelativePath)\lib\%(CopyLocalAssembly.TFM)\%(CopyLocalAssembly.Identity).dll</FullFilePath>
-      </CopyLocalAssembly>
-
-      <!-- Add our tasks assembly -->
-      <CopyLocalAssembly Include="Microsoft.NET.Build.Tasks">
-        <FullFilePath>$(OutDir)\Microsoft.NET.Build.Tasks.dll</FullFilePath>
-        <DestinationTFM>net46;netcoreapp1.0</DestinationTFM>
-      </CopyLocalAssembly>
-
-      <Net46CopyLocalAssembly Include="@(CopyLocalAssembly)"
-                              Condition="$([System.Text.RegularExpressions.Regex]::IsMatch('%(DestinationTFM)', '(^|;)net46($|;)'))" />
-      <NetCoreAppCopyLocalAssembly Include="@(CopyLocalAssembly)"
-                                   Condition="$([System.Text.RegularExpressions.Regex]::IsMatch('%(DestinationTFM)', '(^|;)netcoreapp1.0($|;)'))" />
-
-      <CopyLocalBuildFile Include="$(OutDir)build\**\*.*" />
-      <CopyLocalBuildFile>
-        <RelativePath>build\%(RecursiveDir)</RelativePath>
-      </CopyLocalBuildFile>
-
-      <CopyLocalBuildCrossTargetingFile Include="$(OutDir)buildCrossTargeting\**\*.*" />
-      <CopyLocalBuildCrossTargetingFile>
-        <RelativePath>buildCrossTargeting\%(RecursiveDir)</RelativePath>
-      </CopyLocalBuildCrossTargetingFile>
-
-      <CopyLocalSdkFile Include="$(OutDir)sdk\**\*.*" />
-      <CopyLocalSdkFile>
-        <RelativePath>Sdk\%(RecursiveDir)</RelativePath>
-      </CopyLocalSdkFile>
-
-      <CopyLocalSatelliteAssembly Include="$(OutDir)net46\**\Microsoft.NET.Build.Tasks.resources.dll">
-        <RelativePath>tools\net46\</RelativePath>
-      </CopyLocalSatelliteAssembly>
-      <CopyLocalSatelliteAssembly Include="$(OutDir)**\Microsoft.NET.Build.Tasks.resources.dll" Exclude="@(CopyLocalSatelliteAssembly)">
-        <RelativePath>tools\netcoreapp1.0\</RelativePath>
-      </CopyLocalSatelliteAssembly>
-      <CopyLocalSatelliteAssembly>
-        <RelativePath>%(RelativePath)%(RecursiveDir)</RelativePath>
-      </CopyLocalSatelliteAssembly>
-
-      <CopyLocalBuildFile Include="@(CopyLocalBuildCrossTargetingFile)" />
-      <CopyLocalBuildFile Include="@(CopyLocalSdkFile)" />
-      <CopyLocalBuildFile Include="@(CopyLocalSatelliteAssembly)" />
-
-      <!-- Use the contents of the output directory of the net46 build of the SDK tasks, except
-           for MSBuild and Roslyn DLLs and their dependencies-->
-      <Net46OutputAssembly Include="$(OutDir)net46\*.dll" />
-      
-      <!-- Use the versions of these that come with MSBuild -->
-      <Net46OutputAssembly Remove="$(OutDir)net46\Microsoft.Build.Framework.dll" />
```

Comment:
For net46, ExcludeAssets=Runtime. For netcoreapp, that + FilterCopyLocal.

Basically we adjust the build step to deploy only the right assemblies and then packaging can copy them wholesale.